src/oic/extension/device_flow.py

```diff
@@ -94,7 +94,7 @@ def authorization_request(self, scope=""):
             req["scope"] = scope
 
         http_response = self.host.http_request(
-            self.host.provider_info["device_endpoint"], "POST", req.to_urlencoded()
+            self.host.provider_info["device_authorization_endpoint"], "POST", req.to_urlencoded()
```

Comment:
Could you please update the formatting as suggested by the formater?

Running `black src/ tests/` should be sufficient.